test/functional/api/buffer_spec.lua

```diff
@@ -249,6 +251,17 @@ describe('api/buf', function()
       eq(1, funcs.exists('b:lua'))
       curbufmeths.del_var('lua')
       eq(0, funcs.exists('b:lua'))
+      curbuf('set_var', 'changedtick', true)
```

Comment:
shouldn't this be an error? `:let b:changedtick=42` is an error. Also why set it to boolean instead of an integer?